### ISSUE TITLE
Balloon notifications for processes?

### ISSUE THREAD

**Issue by PsyZeus** [ampache/ampache#3346]
As per title, i don't know if it is broken somehow or removed but i don't get any notifications for processes anymore on ampache 5.5.1 when i add/remove/update etc

for the sake of music BRING IT BACK!!!
i'm feeling like i'm blind, i don't know what it is doing!!

**Comment by lachlan-00** [comment]
I'm not sure why it's not there, not something done by choice

**Comment by lachlan-00** [comment]
there's a JS error to follow at least
![image](https://user-images.githubusercontent.com/1305249/189306652-b7f523f4-e18c-4612-89df-00d918b75761.png)

**Comment by lachlan-00** [comment]
Interestingly develop.ampache.dev doesn't have this issue... it shows me the messages right away

**Comment by SimonHova** [comment]
A bit late to the thread, but I am not sure what you mean by notifications and processes. Can you post a screen shot? I'd love to find out if there's a feature that I'm not using!

**Comment by lachlan-00** [comment]
these are the black popup messaged on worker actions like catalog processing
![image](https://user-images.githubusercontent.com/1305249/189453089-b3f82e79-d2a3-4377-a474-511e9122e6c3.png)

I can't see an obvious reason for them not firing visually as they pop up on the demo sites no problem. The process and the worker are running too.

Interestingly i can't get them to fire at all on my home server but all the demo sites work fine and they're all different branches.

**Comment by PsyZeus** [comment]
@lachlan-00 
sorry for my late response i had a busy weekend!!

i'm pretty sure it started with the update to 5.5.1

so after all your inputs it seems so weird why it doesn't work only in our home servers. ?????
i can test whatever you looking for as i have an instance for testing already, just guide me to what i need to do, happy to help the project as always :D

**Comment by PsyZeus** [comment]
@SimonHova so you never answered, do you get/see those notifications on 5.5.1 ? 
are you on develop or release?

**Comment by lachlan-00** [comment]
i'll set up a bunch of docker sites and test if there's a point i can see it stopping. I definitely changed JS code over recent time and there are probably still many Ampache things that break when you 'fix' something

**Comment by SimonHova** [comment]
I am 5.5.1 develop, and I have seen them, but only during issue #3342 when
it complained that it had lost the database connection.

**Comment by lachlan-00** [comment]
The code is reaching the output function but not always firing. i wonder if a composer dep update has made this happen as the code itself has not changed for years.

src/Module/Util/Ui.php
```
2022-09-13T15:36:25+10:00 [user] (Ampache\Module\Util\Ui) -> update_text SSE_OUTPUT 1
2022-09-13T15:36:25+10:00 [user] (Ampache\Module\Util\Ui) -> update_text SSE_OUTPUT /mnt/files-music/upload/user/YOUR NEW MORNING ALARM.mp3
```

which is this function to print the js output
![image](https://user-images.githubusercontent.com/1305249/189818486-6718cc41-883d-4992-a74f-d76aa27c9571.png)

**Comment by lachlan-00** [comment]
And the errors work but not the onmessage.
![image](https://user-images.githubusercontent.com/1305249/189819527-207d7f91-192f-404a-86c6-cb5361db55c9.png)

I have no idea what is getting sent to the eval but that's probably something important

**Comment by PsyZeus** [comment]
So, i still don't have them with 5.5.2 :(

What about you people? @lachlan-00 @SimonHova

**Comment by lachlan-00** [comment]
yup, they're still missing from a lot of places, i get the end notification on servers with the issue but not the start/finish.
![image](https://user-images.githubusercontent.com/1305249/207975259-1e6feb1a-09a9-49a0-8cfd-a0593bfae79e.png)

I'm still not sure why the worker doesn't fire for some of those messages and not others.

Part of a larger issue is that the catalog actions for web and for cli need to be separated out. i'm sure work done on the cli actions has changed for the sse worker but haven't been able to confirm.
